### PR TITLE
Update adding-dependencies.md

### PR DESCRIPTION
**Pull request by bikramkawan**

### CODE REVIEW

**Comment by bikramkawan** [comment]
@thinkh @sgratzl

File: contributors/adding-dependencies.md [hunk 1]
```diff
@@ -63,6 +63,21 @@ Note, that you must import the specific D3 v4 modules to your TypeScript file as
...
+```json
+"dependencies": {
+   "jquery-ui": "1.12.1"
+  },
+```
+2.[Update your setup](#update-setup)
+
+3.Importing jquery-ui in pycharm
+
+`import 'jquery-ui/ui/widgets/sortable';`
```
Comment (sgratzl) [inline contributors/adding-dependencies.md]: why about the stylesheets of jquery-ui?

File: contributors/adding-dependencies.md [hunk 2]
```diff
@@ -63,6 +63,21 @@ Note, that you must import the specific D3 v4 modules to your TypeScript file as
 ```js
 import {scaleLinear} from 'd3-scale';
 ```
+**Importing jquery-ui**
+
+1.Add the dependencies in package.json as :
+```json
+"dependencies": {
+   "jquery-ui": "1.12.1"
+  },
```
Comment (sgratzl) [inline contributors/adding-dependencies.md]: I would prefer `npm install -save jquery-ui`

File: contributors/adding-dependencies.md (hunk 1 above)
Comment (sgratzl) [reply contributors/adding-dependencies.md]: and that you need jQuery in addition

Comment (thinkh) [reply contributors/adding-dependencies.md]: Done. Please have a look, if the CSS import is correct.

Comment (sgratzl) [reply contributors/adding-dependencies.md]: I'm not sure whether it works, you might have to prefix the import with `style-loader!css-loader!`

Comment (thinkh) [reply contributors/adding-dependencies.md]: Sure. Updated.

**Pull request merged**